packages/lit-dev-content/site/docs/components/context.md

```diff
@@ -0,0 +1,526 @@
+
+This protocol enables interoperability between elements (or even non-element code) regardless of how they were built. Via the context protocol, a Lit-based element can provide data to a consumer not built with Lit, or vice versa.
+
+The Context Protocol is based on DOM events. A consumer fires a `context-request` event that carries the context key that it wants, and any element above it can listen for the `context-request` event and provide data for that context key.
+
+`@lit-labs/context` implements this event-based protocol and makes it available via a few reactive controllers and decorators.
+
+### Context Objects
+
+Contexts are identified by _context objects_ or _context keys_. They are objects that represent some potential data to be shared by the context object identity. You can think of them as similar to Map keys.
+
+### Providers
+
+Providers are usually elements (but can be any event handler code) that provide data for specific context keys.
+
+### Consumers
+
+Consumers request data for specific context keys.
+
+### Subscriptions
+
+When a consumer requests data for a context, it can tell the provider that it wants to _subscribe_ to changes in the context. If the provider has new data, the consumer will be notified and can automatically update.
+
+## Usage
+
+### Defining a context
+
+Every usage of context must have a context object to coordinate the data request. This context object represents the identity and type of data that is provided.
+
+Context objects are created with the `createContext()` function:
+
+```ts
+export const myContext = createContext(Symbol('my-context'));
+```
+
+It is recommended to put context objects in their own module so that they're importable independent of specific providers and consumers.
+
+#### Context type-checking
+
+`createContext()` takes any value and returns it directly. In TypeScript, the value is cast to a typed `Context` object, which carries the type of the context _value_ with it.
+
+In case of a mistake like this:
+```ts
+const myContext = createContext<Logger>(Symbol('logger'));
+
+class MyElement extends LitElement {
+  @provide({context: myContext})
+  name: string
+}
+```
+
+TypeScript will warn that the type `string` is not assignable to the type `Logger`.
+
+#### Context equality
+
+Context objects are used by providers to match a context request event to a value. Contexts are compared with with strict equality (`===`), so a provider will only handle a context request if its context key equals the context key of the request.
+
+This means that there are two main ways to create a context object:
+1. With a value that is globally unique, like an object (`{}`)  or symbol (`Symbol()`)
+2. With a value that is not globally unique, so that it can be equal under strict equality, like a string (`'logger'`) or _global_ symbol (`Symbol.for('logger')`).
+
+If you want two _separate_ `createContext()` calls to referrer to the same
```

Comment:
```suggestion
If you want two _separate_ `createContext()` calls to refer to the same
```

Reply:
👋 
Just dropping in to say hi and correct a typo for old time's sake 😂 ... Apparently I haven't turned off my GitHub notifications yet ...